runtime/src/accounts_db.rs

```diff
@@ -956,6 +961,12 @@ pub struct AccountsDb {
     /// Set of storage paths to pick from
     pub(crate) paths: Vec<PathBuf>,
 
+    ledger_path: PathBuf,
```

Comment:
Can we call it `accounts_hash_cache_path` or something like that? It's not really holding ledger.

Reply:
done